### PR TITLE
use `ip link show` to get virtual bridge interface name.

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

**Comment by kbidarkar** [comment]
Fixed in https://github.com/SatelliteQE/automation-tools/issues/238

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -263,7 +263,7 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
         # Ignore errors related to infiniband if any and just fetch
         # virtual bridge information.
         interface = run(
-            'ifconfig 2> /dev/null | grep virbr | awk \'{print $1}\''
+            'ip link show | grep virbr | awk \'{print $2}\' | head -n1'
```
Comment (sthirugn) [inline automation_tools/__init__.py]: :+1: 

running in my local printed the following as expected:
```
# ip link show | grep virbr | awk '{print $2}' | head -n1
virbr0:
```

**Comment by sthirugn** [comment]
ACK

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -263,7 +263,7 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
...
         )
         # Always select the first interface
         interface = interface.split('\n', 1)[0].strip()
```
Comment (elyezer) [inline automation_tools/__init__.py]: Since head is being used previously, this is not required anymore.

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by kbidarkar** [comment]
Updated, ready for merge.

**Comment by elyezer** [comment]
ACK

**Pull request merged**